tests/entities_item_tests.py

```diff
@@ -220,6 +220,35 @@ def test_item_should_fail_to_update_column_value_with_invalid_column_value(get_b
     item.change_column_value(column_value='5')
 
 
+@patch('moncli.api_v2.get_items')
+@raises(en.item.InvalidParameterError)
+def test_should_fail_to_update_item_name_if_input_new_name_parameter_is_none(get_items):
+
+    # Arrange
+    get_items.return_value = [{'id': 2, 'name': 'Test Item 2'}]
+    item = client.get_items()[0]
+
+    # Act
+    item.change_item_name(None)
+
+
+@patch.object(en.Item,'change_item_name')
```

Comment:
This is not the correct all to mock.  

The correct call is for moncli.api_v2.change_multiple_column_value, as it is being called inside of this Item method that you are trying to test.